packages/fast-components-react-msft/src/text-area/examples.data.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from "react";
+import { ComponentFactoryExample } from "@microsoft/fast-development-site-react";
+import { TextArea, TextAreaProps } from "./index";
+import schema from "@microsoft/fast-components-react-base/dist/text-area/text-area.schema.json";
+import Documentation from "./.tmp/documentation";
+
+export default {
+    name: "Text Area",
+    component: TextArea,
+    schema: schema as any,
+    documentation: <Documentation />,
+    detailData: {
+        placeholder: "Textarea placeholder text",
+    },
+    data: [
+        {
+            placeholder: "Text area placeholder text",
+        },
+        {
+            disabled: true,
+            placeholder: "Disabled text area text",
```

Comment:
Should we show a read-only example?

Reply:
Per other comment, read only needs to be *removed* from the schema

Reply:
Got it